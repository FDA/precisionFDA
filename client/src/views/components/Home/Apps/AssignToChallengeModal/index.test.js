import React from 'react'
import { shallow } from 'enzyme'

import { AssignToChallengeModal } from '.'


describe('AssignToChallengeModal test', () => {
  it('should render', () => {
    const component = shallow(<AssignToChallengeModal />)

    expect(component).toMatchSnapshot()
  })
})
