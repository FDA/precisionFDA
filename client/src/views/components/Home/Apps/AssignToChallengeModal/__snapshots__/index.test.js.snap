// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AssignToChallengeModal test should render 1`] = `
<div
  className="objects-actions-modal"
>
  <Modal
    hideModalHandler={[Function]}
    modalFooterContent={
      <React.Fragment>
        <Button
          onClick={[Function]}
        >
          Cancel
        </Button>
        <Button
          disabled={true}
          onClick={[Function]}
          type="primary"
        >
          Assign
        </Button>
      </React.Fragment>
    }
    noPadding={true}
    title="Assign to Challenge"
  >
    <div
      className="home-page-layout__modals-items"
    />
  </Modal>
</div>
`;
